packages/web-embed-api/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@formsort/web-embed-api",
-  "version": "2.2.1",
+  "version": "2.2.2",
```

Version bumping happens automatically when you release the package so please don't do it manually.
```suggestion
  "version": "2.2.1",
```